.github/workflows/python-coverage.yaml

```diff
@@ -0,0 +1,32 @@
+# https://github.com/marketplace/actions/python-coverage
+
+name: coverage
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  container-unit-test:
+    runs-on: [self-hosted, docker]
+    container:
+      image: localhost:5000/flag-gems-ci:v1.0
+      ports:
+        - 81
+      options: --gpus all --hostname flag-gems_cicd_ut
+    steps:
+      - name: check-gpu-free
+        run: tests/scripts/gpu_check.sh
+
+      - name: Run pytest
+        run: CUDA_VISIBLE_DEVICES=0 coverage run -m pytest -s tests/test_binary_pointwise_ops.py::test_accuracy_add --cov-report xml:coverage.xml
```

Comment:
Is this a coverage test for add only?

Reply:
No, currently I just take `add` for the test. I must push my code ro make CI run to test the result, so this PR is not ye t ready, but I cannot mark it draft because a draft PR will not make CI run